include/matx/core/print.h

```diff
@@ -149,7 +149,25 @@ namespace matx {
           fprintf(fp, ", ");
       }
 
-      if constexpr (is_tensor_view_v<Op>)
+      if constexpr (is_sparse_tensor_v<Op>)
```

Comment:
Should this also change line 143 to either display Sparse Tensor, Dense Tensor, or Operator?

Reply:
Good idea. Done.